src/app.py

```diff
@@ -2321,6 +2348,25 @@ def get_prov_info():
         internal_dict[HEADER_DATASET_CREATED_BY_EMAIL] = dataset['created_by_user_email']
         internal_dict[HEADER_DATASET_DATE_TIME_MODIFIED] = datetime.fromtimestamp(int(dataset['last_modified_timestamp']/1000.0))
         internal_dict[HEADER_DATASET_MODIFIED_BY_EMAIL] = dataset['last_modified_user_email']
+        internal_dict[HEADER_DATASET_LAB_ID] = dataset['lab_dataset_id']
```

Comment:
Do you need to check the existence of `lab_dataset_id` before assigning it? I don't think this field is always available and we'll get KeyError exception when it's not set.

https://github.com/hubmapconsortium/entity-api/blob/Derek-Furst/prov-info/src/schema/provenance_schema.yaml#L250